src/main/java/com/mcmoddev/bot/handlers/CommandHandler.java

```diff
@@ -46,6 +35,7 @@ public CommandHandler () {
             registerCommand("members", new CommandMemberCount());
             registerCommand("server", new CommandServerInfo());
             registerCommand("curse", new CommandCurse());
+            registerCommand("xy", new CommandXY());
```

Comment:
I feel like this should be in caps, seeing as it is "the XY problem", not "the xy problem".

Reply:
We must register the key in lowercase but can then use it in lower or uppercase. See [CommandHandler Ln 119] https://github.com/MinecraftModDevelopment/MMDBot/blob/master/src/main/java/com/mcmoddev/bot/handlers/CommandHandler.java#L119)